src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/DistributedPubSubMediator.cs

```diff
@@ -157,10 +155,10 @@ public DistributedPubSubMediator(DistributedPubSubSettings settings)
             if (!string.IsNullOrEmpty(_settings.Role) && !_cluster.SelfRoles.Contains(_settings.Role))
                 throw new ArgumentException($"The cluster member [{_cluster.SelfAddress}] doesn't have the role [{_settings.Role}]");
 
-            //Start periodic gossip to random nodes in cluster
-            _gossipCancelable = Context.System.Scheduler.ScheduleTellRepeatedlyCancelable(_settings.GossipInterval, _settings.GossipInterval, Self, GossipTick.Instance, Self);
```

Comment:
LGTM (should probably keep the comment though)

Reply:
the comment was moved to the `PreStart()` override where we start the periodic timer